### PR TITLE
wk10d1hw

### PR DESCRIPTION
**Pull request by leenahabi**

### CODE REVIEW

File: db/schema.js [hunk 1]
```diff
@@ -2,3 +2,37 @@ const mongoose = require('mongoose');
...
+
+let MenuItems = new Schema ({
+    title: String
+})
+
+let Restaurant = new Schema ({
+    name: String,
+    adress: {street:String,zipcode:Number},
+    yelpUrl:String,
+    items:[MenuItems]
```
Comment (Salman778) [inline db/schema.js]: items: [{ type: Schema.Types.ObjectId, ref: 'Menu' }]

File: app.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+//>>>
+// create 
+//works except createMenu
+const createMenu = (title) => {
+    const  menuParams = {
+        title:title
+    }
+    let newMenuItem = new Menu(menuParams);
+     newMenuItem.save()
```
Comment (Salman778) [inline app.js:30]:
```js
 const newItem = newMenuItem.save()
 return newItem
```

File: app.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+        name : name,
+        address:{
+            street:street,
+            zipcode:zipcode
+        },
+        yelpUrl:yelpUrl,
+        items:[]
+    }
+    let newRest = new Restaurant(restuarantParams)
+    const Menuitem = createMenu(item)
```
Comment (Salman778) [inline app.js:44]:
```js
const Menuitem = createMenu(item)
    .then(
        item => {
            newRest.items.push(item)
            newRest.save()
                .then(restuarant => {
                    console.log(restuarant)
                })
                .catch(err => {
                    console.log(err)
                })
                .then(done)
        } 
    )
```

**Comment by leenahabi** [comment]
it doesn't work but i tried :(

**Pull request closed without merging**